src/components/dar_drafts_table/DarDraftTable.js

```diff
@@ -243,11 +243,10 @@ export default function DarDraftTable(props) {
     }),
     h(ConfirmationModal, {
       showConfirmation,
-      styleOverrise: { height: '35%' },
       closeConfirmation: () => setShowConfirmation(false),
       title: 'Delete Draft DAR',
       message: `Are you sure you want to delete DAR draft ${getIdentifier({id: selectedDraft.id, data: selectedDraft.data})}`,
-      header: getModalHeader(),
+      header: getModalHeader,
```

Comment:
I'm realizing now that the example I was looking at with "Title" as the header was actually for a DAR draft named "Title" (sorry about that!). Now I'm not seeing a header at all, so I think the original code was correctly making the header the name of the draft

Reply:
Hah, will revert that change then!